packages/button/src/ButtonBase.ts

```diff
@@ -76,25 +85,17 @@ export class ButtonBase extends ObserveSlotText(LikeAnchor(Focusable), '', [
         });
     }
 
-    public override click(): void {
```

Comment:
This event is only firing when with a programatic `click()`--like in parent components or in the majority of our tests. It is not firing with a mouse click (or, by extension, a VoiceOver click (i.e. <kbd>CTRL+Option+Space</kbd>. I removed it  and made sure this functionality was in `handleClickCapture()` function which is called by the `click` event listener in the `constructor()`, which is fired by programatic and mouse clicks.